packages/replay-next/components/sources/utils/classNameToTokenTypes.ts

```diff
@@ -0,0 +1,7 @@
+import { TokenType } from "../../lexical/plugins/code/types";
+
+export default function classNameToTokenTypes(className: string): TokenType[] {
+  // e.g. "tok-comment" -> ["comment"]
+  // e.g. "tok-variable tok-definition" -> ["variable", "definition"]
```

Comment:
This was a case we weren't properly handling before:
```js
let foo = "abc";
```

In the above example, `foo` is both a variable _and_ a definition.